lib/roger/server.rb

```diff
@@ -63,6 +63,17 @@ def use(*args, &block)
       @stack.use(*args, &block)
     end
 
+    # Pass options to the Roger rack application
+    # Calling run is entirely optional and doesn't change
```

Comment:
Nope; but callling it `options` is also weird. It's currently in line with with the processor does. Alternatively we could call it `application_options=()`